### DISCUSSION TITLE
apps in huggingface?

### DISCUSSION THREAD

**Discussion by Jose-jme** [huggingface/huggingface.js#951] [Q&A]
Hi, does anyone know why Ollama doesn't appear in the local apps section on huggingface? Although I think the apps that appear are incredible, in fact some are better in specific aspects.

**Comment by coyotte508** [comment]
https://github.com/huggingface/huggingface.js/pulls?q=sort%3Aupdated-desc+is%3Apr+ollama you can check the comments in  those PRs
